netebpfext/net_ebpf_ext.c

```diff
@@ -696,8 +698,9 @@ static ebpf_error_code_t
 _net_ebpf_ext_program_information_encode_xdp()
 {
     ebpf_error_code_t return_value;
-    uint8_t* buffer = NULL;
+    const uint8_t* buffer = NULL;
     unsigned long buffer_size = 0;
+#if defined(MSRPC_LIB)
```

Comment:
question: when is this defined vs not defined?  Is this file compiled twice?

Reply:
Dead code I guess. This was place-holder until I got it to bind to msrpc.sys. Given that seems like a no-go I will remove this code.